.vscode/settings.json

```diff
@@ -5,5 +5,8 @@
 	"editor.codeActionsOnSave": {
 		"source.fixAll": true,
 		"source.organizeImports": false
+	},
+	"files.exclude": {
+		"node_modules": true
```

Comment:
Up to crawl, but I don't think this should be added

Reply:
This was added just to make it easier on my end, was going to remove it (and revert the collection dep thing) once everything was good. 👍

Reply:
Personally I don't think it hurts to have, but I'll let Crawl decide too :D